site/cds_rdm/communities/views.py

```diff
@@ -0,0 +1,167 @@
+# -*- coding: utf-8 -*-
```

Comment:
A bit of more ❤️ required here ☺️

Reply:
agreed. If you don't mind, I would merge this one and work on the theming shortly after migration - for a bit of time the community may be not themed. WDYT?